docs/theming/developers.md

```diff
@@ -200,46 +207,47 @@ themable graphics
 
 ## How theming works
 
-We developed our theming system with web standards in mind, aiming for it to use
-CSS over JavaScript as much as possible. At the current state of the art, we
-found that some JavaScript is required for the system to work correctly, but in 
-the near  future, we expect to be able to remove all or nearly all of that 
-JavaScript, and have the theming system work entirely (or almost entirely) 
+We developed our theming system with web standards in mind, aiming to use
+CSS over JavaScript as much as possible. At the current state of browser technology,
+we found that some JavaScript is required for the system to work correctly.
+But shortly, we expect to be able to remove all or nearly all of that 
+JavaScript and have the theming system work entirely (or almost entirely) 
 through plain CSS. The following explains how the system currently operates, how
 element authors can hook into it, and what the future holds for theming.
 
 ### Current implementation - Context Protocol
 
-<abbr title="Red Hat Design System">RHDS</abbr>' Theming system is primarily 
-about styles. Tt currently relies on JavaScript to work, by way of the [context 
+<abbr title="Red Hat Design System">RHDS</abbr>’ theming system is primarily 
+about styles. It currently relies on JavaScript to work, via the [context 
 protocol][contextprotocol] developed by the web components community to support 
 passing data between components.
 
 Our system utilizes this protocol with the setting of the `color-palette` 
 attribute on a provider element which makes its context data (in our case, 
-`light` or `dark`) to it's children.  By doing so we can ensure accessible 
+`light` or `dark`) to its children. By doing so we ensure accessible 
```

Comment:
I feel like a word is missing before "to its children." Is "available" the appropriate word here?

Reply:
[a772217](https://github.com/RedHat-UX/red-hat-design-system/pull/1937/commits/a772217d3dc68bee1fa79a0781d8b31ed8b69886)